.github/workflows/ci.yml

```diff
@@ -52,5 +52,10 @@ jobs:
           https://install.lix.systems/lix | sh -s -- install --no-confirm
 
       - name: Check ${{ matrix.name }}
-        run:
-          nix build -L --log-lines 0 .#checks.x86_64-linux.${{ matrix.check }}
+        run: |
+          sudo tee -a /etc/nix/nix.conf > /dev/null <<EOF
+          extra-substituters = https://d17qv3gttz4z4k.cloudfront.net
+          extra-trusted-public-keys = gglite-nix-cache-1:Zdz1mEqn//xa8ORxHkc76auwxmX8/6C2K/zWRjmq8Co=
```

Comment:
What is the cache isn't signed by this key? What is the process for signing the cached item?